docs/index.rst

```diff
@@ -39,16 +45,7 @@ Links
 
 License: BSD-3
 
-.. _astroplan_news:
 
-News
-====
-
-* August 28, 2015: Astroplan **0.1** release
```

Comment:
The places you mention where to find the version number (like setup.py) are very hard to find for **users**.

When I discover a new package or software tool I want to use one of my first questions is "is this an active project?" and if I come back to a package or tool I haven't used in a while "is there a new release?" are "are there any new features?".

So I think it's good to have that info on the front page of the project.
Just as one example, see http://getbootstrap.com/ which has "Currently v3.3.5" front and center, and then "version 4 is coming" at the top.

But again ... whatever you decide to put here is fine with me ... let's try to assemble the 0.1 release and we can always improve / re-discuss the front page for 0.2 or 1.0.